africanlii/views/home.py

```diff
@@ -35,6 +35,18 @@ def get_context_data(self, **kwargs):
         context["recs"] = RegionalEconomicCommunity.objects.prefetch_related("locality")
         context["member_states"] = MemberState.objects.prefetch_related("country")
         context["taxonomies"] = Taxonomy.get_tree()
+        # We explicitly exclude African Commission on Human and Peoples Rights (ACHPR) and
+        # African Committee of Experts on the Rights and Welfare of the Child (ACERWC)
+        excluded_court_codes = ["ACHPR", "ACERWC"]
```

Comment:
The admin now has control over what courts are shown, by deciding what courts go under the various court classes. So this check can fall away, and you can simply show all the court classes that have courts in them.